languagetool-language-modules/eo/src/main/resources/org/languagetool/rules/eo/grammar.xml

```diff
@@ -4047,6 +4080,16 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
         <example correction="simplan|facilan">Li donis <marker>malkomplikan</marker> respondon.</example>
       </rule>
     </rulegroup>
+    <rulegroup id="HUMIDA" name="humida → malseka">
+      <rule>
+        <pattern>
+          <marker><token inflected="yes" regexp="yes">(ne|mal)?humid((umi|igi|iĝi)|(a|o|eco|ega|eta)j?n?)</token></marker>
```

Comment:
Since you wrote inflected="yes", the rule should work without having to put j?n?  at the end of the regexp.

The regexp does not look right.  Better would be:

<token …>(ne|mal)?humid(um|ig|iĝ|ec|et|eg)[ao]</token>